cmd/ore/aws/upload.go

```diff
@@ -236,6 +238,21 @@ func runUpload(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if len(uploadTags) > 0 {
+		hvmTagMap := make(map[string]string)
+		for _, tag := range uploadTags {
+			splitTag := strings.Split(tag, "=")
+			key, value := splitTag[0], splitTag[1]
+			hvmTagMap[key] = value
+		}
+		err = API.CreateTags([]string{hvmID}, hvmTagMap)
```

Comment:
This does not tag the underlying snapshot.  Is that intentional?

Reply:
@bgilbert: Is there a particular reason we'd want to tag the snapshot?

Reply:
Deleting an AMI doesn't automatically delete the underlying snapshot, resulting in a stray snapshot without much context about why it exists.  So I'd be in favor of tagging the snapshot as well, unless there's a reason to do otherwise.

Reply:
@sayanchowdhury Any progress?

Reply:
@ashcrow @sayanchowdhury is on vacation. @mike-nguyen picked it up in https://github.com/coreos/mantle/pull/873